hack/testdata/multi-resource-yaml-modify.yaml

```diff
@@ -19,6 +19,8 @@ metadata:
   name: mock
 spec:
   replicas: 1
+  selector:
```

Comment:
Otherwise the selectors are changing in test-cmd.sh (https://github.com/kubernetes/kubernetes/blob/master/hack/test-cmd.sh#L1964), so new pods are created after "kubectl replace" and "kubectl edit". We are lucky all those pods get deleted after the test. They are deleted only because the RC created [later](https://github.com/kubernetes/kubernetes/blob/master/hack/test-cmd.sh#L1970) happens to have the same selector.